app/models/region.rb

```diff
@@ -1,13 +1,17 @@
 class Region < ActiveRecord::Base
-  validates_uniqueness_of :slug
   validates_uniqueness_of :name
 
   has_many :region_organizers
   has_many :organizers, :through => :region_organizers, :source => :user
+  has_many :region_slugs, dependent: :destroy
 
   scope :active, -> { where(active: true) }
 
   def to_param
-    "#{self.id}-#{self.slug}"
+    "#{self.id}-#{self.main_slug}"
+  end
+
+  def main_slug
+    region_slugs.try(:first).try(:slug)
```

Comment:
I'm not entirely sure if the selection of the main slug is stable. This determines on the order in which the slugs are returned by the database, right? How could I choose which slug is the main one?